src/main/fc/cli.c

```diff
@@ -302,9 +302,36 @@ static void printValuePointer(const clivalue_t *var, const void *valuePointer, b
 {
     if ((var->type & VALUE_MODE_MASK) == MODE_ARRAY) {
         for (int i = 0; i < var->config.array.length; i++) {
-            uint8_t value = ((uint8_t *)valuePointer)[i];
+            switch (var->type & VALUE_TYPE_MASK) {
+                default:
+                case(VAR_UINT8): {
```

Comment:
Some minor coding style points. BF style is to align `case` with `switch`, and have no brackets around case item. Also braces within `case` statements are indented and below `case` keyword, but are better avoided if possible. So
```
    switch (var->type & VALUE_TYPE_MASK) {
    default:
    case VAR_UINT8:
        // fetch data
        cliPrintf("%d", ((uint8_t *)valuePointer)[i]);
        break;
```